docs/dynamic-programming/equal-sum-partition.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Equal Sum Partition
+slug: /dynamic-programming/equal-sum-patition
+---
+
+## Introduction
+
+We have to determine if the given array of positive integers can be divided into two subsets which need not be equal such that the sum of elements of both the subsets is equal. Each element must belong to exactly one subset only.
```

Comment:
Add a bit more description. Right now, it won't make any sense to a reader who might not have came across such a problem.